### PR TITLE
Remote Snarking Plugin

### PR DESCRIPTION
**Pull request by Bind**
Screen Shot 2021-01-19 at 10 02 37 PM

Run snark proof generation on a remote server & specify how many snarks in parallel to run.

### CODE REVIEW

**Comment by Bind** [comment]
@phated Using **A LOT** of your queued-move stuff here. I think I need some of the internals a bit more exposed but wanted to push this up as a starting point! 😄

**Comment by phated** [comment]
❤️

**Comment by phated** [comment]
@Bind I was going to merge this but it looks like it doesn't have an index file to show up on the website?

**Comment by Bind** [comment]
Whoops will fix

**Comment by Bind** [comment]
@phated should be good to go

File: content/productivity/remote-snarker/index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: Remote Snarker
+date: 2020-1-20
```
Comment (phated) [inline content/productivity/remote-snarker/index.md:3]:
```suggestion
date: 2020-01-20
```

checking to see if this 0 changes sort order for some reason.

File: content/productivity/remote-snarker/index.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: Remote Snarker
+date: 2020-01-20
```
Comment (phated) [inline content/productivity/remote-snarker/index.md:3]: Oh, it's 2021!
```suggestion
date: 2021-01-20
```

**Pull request merged**